nvi

```diff
@@ -149,13 +149,24 @@ install_node() {
 
   cd "$INSTALL_DIR/$PACKAGE"
   # Install executables in the bin dir.
-  cp -- "bin/node" "$BIN_DIR/node"
-  ln \
+  if ln \
     --force \
     --symbolic \
     -- \
-    "$PWD/bin/npm" \
-    "$BIN_DIR/npm"
+    "$INSTALL_DIR/$PACKAGE/bin/node" \
+    "$EXEC_DIR/node"
+  then
+    ln \
+      --force \
+      --symbolic \
+      -- \
+      "$INSTALL_DIR/$PACKAGE/bin/npm" \
+      "$EXEC_DIR/npm"
+  else
+    fatal "Couldn't write to symlink \"$EXEC_DIR/node\". Possibly a permission issue. Aborting."
+  fi
+
+  echo "Successfully "
```

Comment:
I see now that I never finished this line. It was supposed to mention the successfully installed version.